frontend/src/pages/miscellaneous/ProfilePage.tsx

```diff
@@ -67,8 +69,16 @@ export default function ProfilePage() {
         async function fetchUserData() {
             const username = localStorage.getItem('username') || '';
             const email = localStorage.getItem('email') || '';
+            const profileImage = localStorage.getItem('profileImage') || '';
             setUsername(username);
             setEmail(email);
+
+            // TODO: Change Default value and host a default URL
+            if (profileImage == "NaN") {
```

Comment:
Would it be more efficient to set the default image during account creation on the registration page? This way, the image is set once and saved in the database, rather than requiring the app to select the default image every time they visit the profile page. If a user prefers to keep the default image, the app shouldn't have to search for it repeatedly. Additionally, having the check for the default image on the profile page seems unnecessary in this case.